spec/requests/affiliates_spec.rb

```diff
@@ -4,7 +4,7 @@
   describe "GET /affiliates" do
     it "works! (now write some real specs)" do
       get affiliates_path
-      expect(response).to have_http_status(200)
+      expect(response).to have_http_status(302)
```

Comment:
Currently the spec can't use sign_in_auth because it can't find the module ControllerHelpers. I will figure out how to #include the path of the module tomorrow.